src/utils/env.ts

```diff
@@ -176,23 +187,75 @@ export const parseSpacesEnv = (spacesEnv: string): ParsedSpace[] => {
   return spacesEnv.split(",").map((pair) => {
     const [ens, startingBlock, moduleAddress] = pair.split(":");
     return {
-      ens,
+      ens: ens.toLowerCase(),
       startBlock: BigInt(startingBlock),
       moduleAddress: moduleAddress as Address,
     };
   });
 };
 
+type PerEnsRecipients = {
+  common: string[];
+  [ens: string]: string[];
+};
+
 /**
- * Parse the EMAIL_TO environment variable into an array of emails
+ * Parse the EMAIL_TO environment variable into an array of emails.
+ *
+ * The EMAIL_TO consist of comma separated entries that follow one of the following
+ * formats:
+ *  - `foo@bar.com`: A simple email address. It will receive all notifications.
+ *  - `example.eth:foo@bar.com`: A ENS-scoped email address. It will only receive notifications
+ *  for the given space (`example.eth`).
  *
  * @param emailToEnv - The EMAIL_TO environment variable string
  * @returns An array of emails
  *
  * @example
  * const emailTo = parseEmailToEnv("foo@bar.com,bar@foo.com");
  */
-export const parseEmailToEnv = (emailToEnv: string | undefined): string[] => {
-  if (!emailToEnv) return [];
-  return emailToEnv.split(",");
+export const parseEmailToEnv = (emailToEnv: string | undefined): PerEnsRecipients => {
+  const result: PerEnsRecipients = {
+    common: [] as string[],
+  };
+
+  if (!emailToEnv) return result;
+
+  const entries = emailToEnv.split(",");
+
+  entries.forEach((entry) => {
+    const [email, ens] = entry
+      .split(":")
+      // Reversed so the email is always the first part and the ens an
```

Comment:
Because that way I can assume that the first part is the email in both cases, so it will be `[email, undefined]` and `[email, ens]`. Otherwise, it would have to check if there is one or two parameter to assign the email to one or the other. The code after reversal is simpler.

Reply:
@wadader if it is still confusing please tell, not sure how well I've explained myself

Reply:
I understand now. I missed the `.reverse() as [string] | [string, string];`  on line **231** because the comment in between led me to believe this code block was complete

Reply:
@wadader that is fair. Do you think I should place the comment on top of the statements instead of mid-chain?

Reply:
Very Low Priority

Reply:
@wadader done!